src/utils/format.ts

```diff
@@ -0,0 +1,38 @@
+// ref: https://stackoverflow.com/questions/10420352/converting-file-size-in-bytes-to-human-readable-string
```

Comment:
Do we want to use a library for this? Though I guess it's so small it's fine if it works.
https://www.npmjs.com/package/pretty-bytes